### PR TITLE
ui: add bar chart for node list memory & capacity

### LINKED ISSUE #29373: ui: node list: enhance HW metrics totals & percentages

### PR DESCRIPTION
**Pull request by couchand**
Adds bar charts to capacity and memory columns on the node list.

new-node-list

Closes #29373

### CODE REVIEW

**Comment by vilterp** [comment]
Seeing this in Chrome

image

and this in Firefox

image

Oy

File: pkg/ui/src/views/cluster/containers/nodesOverview/barChart.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      .text(Bytes(props.usable));
+
+    const totalNode = total.node();
+    const totalWidth = !totalNode ? 100 : (totalNode as SVGSVGElement).getBBox().width;
+
+    total.attr("x", chartWidth - totalWidth - 0);
+  }
+
+  render() {
+    console.log("foobar!");
```
Comment (vilterp) [inline pkg/ui/src/views/cluster/containers/nodesOverview/barChart.tsx]: stray console.log

**Review by vilterp — COMMENTED** [review]
One nit, as well as the FF vs Chrome issue. Otherwise LGTM.

File: pkg/ui/src/views/cluster/containers/nodesOverview/barChart.tsx (hunk 1 above)
Comment (couchand) [reply pkg/ui/src/views/cluster/containers/nodesOverview/barChart.tsx]: foobar!

**Comment by couchand** [comment]
@vilterp I'm not seeing that in Chrome on Linux, I'll take a look Monday.

**Comment by vilterp** [comment]
Hm looking ok now…
image
Maybe it was a fluke, or only happens with single digits? Idk.

Overall this is looking really nice!

**Pull request merged**